plugin.py

```diff
@@ -787,19 +794,19 @@ def update(self):
         late_by = 0
 
         if layer.providerType() == "spatialite":
-            late_by = versioning.late(
-                    uri.database(), self.pg_conn_info() )
+            late_by = self.spversioning.late(
+                    [uri.database(), self.pg_conn_info()] )
         else: # postgres
-            late_by = versioning.pg_late(
-                    self.pg_conn_info(), uri.schema() )
+            late_by = self.pgversioning.late(
+                    [self.pg_conn_info(), uri.schema()] )
```

Comment:
Those switch could be removed if we go for the abstract `self.versioning` member.